akka-management/core/src/main/scala/com/lightbend/lagom/internal/akka/management/AkkaManagementTrigger.scala

```diff
@@ -45,12 +51,19 @@ private[lagom] class AkkaManagementTrigger(
   /**
    * Starts Akka HTTP Management ignoring the `lagom.akka.management.enabled` setting.
    */
-  private[lagom] def forcedStart(): Future[Done] = {
+  private[lagom] def forcedStart(requester: String): Future[Done] = {
 
     if (!enabled) {
-      // TODO: trace that flag is ignored and AHM is started anyway
+      logger.warn(
+        s"'lagom.akka.management.enabled' property is set to '$enabledRenderedValue', " +
+        s"but Akka Management is being required to start by: '$requester'.")
```

Comment:
Here it makes sense to send a warning. 

The user explicitly set it to `off`, but we don't honour it because some downstream module (Lagom Cluster Bootstrap) requires it.